### PR TITLE
Erivas28 patch 1

### PR DESCRIPTION
**Pull request by erivas28**
outline for portfolio

### CODE REVIEW

**Review by ChenMeng-SPS22 — CHANGES_REQUESTED** [review]
See my pm.

**Comment by erivas28** [comment]
I believe I deleted the excessive files. Let me know if you need me to do anything else, thanks :)

**Comment by ChenMeng-SPS22** [comment]
> 

I think you should remove the entire node_modules/ which contains all the node.js source code.

**Comment by erivas28** [comment]
Just did that Chen, let me know if there is anything else :)

**Comment by erivas28** [comment]
I am done for Week 1 now Chen, I think this is good for presenting tomorrow. I will work on Week 2 later tonight. :)

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,17 +12,14 @@
...
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+// Event for clicking sections
+(function () {
```
Comment (ChenMeng-SPS22) [inline portfolio/src/main/webapp/script.js:16]: Not sure why you define a function and call it immediately. But if this works it's fine.

**Review by ChenMeng-SPS22 — APPROVED** [review]

**Pull request merged**